build_runner/lib/src/entrypoint/options.dart

```diff
@@ -399,3 +418,24 @@ Set<BuildDirectory> _parseBuildDirs(ArgResults argResults) {
   }
   return result;
 }
+
+/// Returns build filters parsed from [buildFilterOption] arguments.
+///
+/// These support `package:` uri syntax as well as regular path syntax,
+/// with glob support for both package names and paths.
+List<BuildFilter> _parseBuildFilters(
+    ArgResults argResults, String rootPackage) {
+  var filterArgs = argResults[buildFilterOption] as List<String>;
+  if (filterArgs?.isNotEmpty ?? false) return null;
```

Comment:
I think this is supposed to be:
```suggestion
  if (filterArgs?.isEmpty ?? true) return null;
```
Making that change locally got me up and running.

Reply:
Heh, nice catch.